Updates owasp java html sanitizer to 20240325.1

joaofreitas-qubit opened the pull request:
Relates with: #qubIT-Omnis-5704

joaofreitas-qubit committed:
Updates owasp java html sanitizer to 20240325.1

Relates with: #qubIT-Omnis-5704

pabrantes commented on bennu-toolkit/pom.xml:
```diff
@@ -138,7 +138,7 @@
 			<artifactId>owasp-java-html-sanitizer</artifactId>
-			<version>r239</version>
+			<version>20240325.1</version>
 			<exclusions>
```
@joaofreitas-qubit No need to keep this exclusion anymore, checking the [pom](https://repo1.maven.org/maven2/com/googlecode/owasp-java-html-sanitizer/owasp-java-html-sanitizer/20240325.1/owasp-java-html-sanitizer-20240325.1.pom) we can see that the jsr305 is now in **provided** scope, which doesn't get downloaded via transitivty.

pabrantes reviewed (changes requested):
Remove exclusion from dependency

joaofreitas-qubit committed:
Removes unnecessary exclusion

This artifact has the provided scope in the new artifact version so
it will not be downloaded by transitivity
Relates with: #qubIT-Omnis-5704
